hydra-core/spec/search_builders/search_builder_spec.rb

```diff
@@ -1,14 +1,13 @@
 require 'spec_helper'
 
-describe SearchBuilder do
-  let(:processor_chain) { [:add_access_controls_to_solr_params] }
-  let(:context) { double('context') }
+RSpec.describe Hydra::SearchBuilder do
+  let(:context) { CatalogController.new }
   let(:user) { double('user', user_key: 'joe') }
   let(:current_ability) { double('ability', user_groups: [], current_user: user) }
   let(:search_builder) { described_class }
 
   subject do
-    search_builder.new(processor_chain, context)
+    search_builder.new(context).with_ability(current_ability)
```

Comment:
If this change in usage of search builders is required then it feels like a backwards-incompatible change requiring a major version.  I think this could be okay, but do you know if the old usage is still possible with Blacklight 7?  Is there way to wrap the search builder in a way to maintain backwards compatibility?

Reply:
I see now that SearchBuilder#initialize can take either 1 or 2 arguments so I think this is still backwards compatible.